docs/release-notes/2.34.0.md

```diff
@@ -0,0 +1,84 @@
+
+# Percona Monitoring and Management 2.34.0
+
+
+| **Release date:** | January 24, 2022                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+
+
+## Release Highlights
+
+### Various UX enhancements to Backup Management
+
+ With this release, we are updating the Backup and Restore functionality with key usability enhancements, including:
+
+- Filtering on services at backup creation time. This makes it considerably easier to find the right service for a backup when working with multiple servers.
+- Automatic redirection to the **Restores** page after initiating a restore. From here you can immediately monitor the restore process, and see the restore history for your PMM instance.
+- Added information about the type of storage for the backup artifacts on the  **All backups** page. This helps you get a more comprehensive overview of your current backups.
+- Design and usability improvements on the **Create Backup** and **All Backups** page to increase consistency with other PMM UI controls.
+
+
+### DBaaS migration
+
+Starting with PMM 2.34.0, PMM installs and manages operators using Operator Lifecycle Manager (OLM) and database clusters using `dbaas-operator`. Nonetheless, these changes are backward incompatible, and you must perform these additional steps:
+
+1. Unregister the Kubernetes cluster from PMM and register it back. PMM will install OLM and `dbaas-catalog` to manage the lifecycle of PXC, PSMDB, and DBaaS operators.
+2. Create a database cluster for each database cluster you have created in PMM. You can use the [script]() to do it automatically.
+
+For more information, see the [architecture]() of DBaaS.
+
+### New components
+
+VMProxy version 2.34.0 has been introduced as part of PMM 2.34.0.
```

Comment:
I think we should rephrase this. Maybe that we just have a new component VMProxy. But also would be cool to see a small description why it is needed. @michal-kralik @dygksy